zh-TW/index.html

```diff
@@ -149,7 +149,8 @@ <h3>Pull Request 檢查清單</h3>
                     <li>
                         如果您的補丁沒有得到審核，或者您需要特定的人來審核， 您可以 @ 請求審核人員在 pull 請求或評論中要求審核， 您也可以在 irc.mozilla.org的 #servo 上要求審核 。 如需中文服務請聯絡：
                         <a href="https://github.com/KiChjang">KiChjang</a> 和
-                        <a href="https://github.com/shinglyu">shinglyu</a>
+                        <a href="https://github.com/shinglyu">shinglyu</a> ，或是在
+                        <a href="https://t.me/rust_tw">Rust 台灣社群</a> 上發問。
```

Comment:
Hmm... this sounds like we'll be splitting the community into different language groups. I'm not sure if that's the best way forward...

Reply:
This Page is Tranditional Chinese version, so I think this is OK? Although Hong Kong also use Tranditional Chinese, but I think it's not a bit deal. If there is a Hong Kong group, we could also add new link!

Reply:
@KiChjang By language group you mean programming language?

Reply:
I just found this is not a best place to add the link, since it is under "when send PR".
What about add a new block, "contact community"?

Reply:
@shinglyu No, I meant that this would encourage a Chinese-speaking community that is separate from the official servo team, and I am uncertain that this is the direction that we want to head towards.